atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -216,6 +227,8 @@ def run(self):
         # changing dockerfile.labels writes out modified Dockerfile - err on
         # the safe side and make a copy
         self.add_aliases(base_image_labels.copy(), dockerfile.labels.copy(), self.labels.copy())
+        if self.info_url_format:
+            self.add_info_url(base_image_labels.copy(), dockerfile.labels.copy(), self.labels.copy())
```

Comment:
we are using it but we need all labels, which are as well in dockerfile and from base_image